src/Symfony/Component/VarDumper/Caster/IntlCaster.php

```diff
@@ -34,4 +34,79 @@ public static function castMessageFormatter(\MessageFormatter $c, array $a, Stub
 
         return $a;
     }
+
+    public static function castNumberFormatter(\NumberFormatter $c, array $a, Stub $stub, $isNested)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'locale' => $c->getLocale(),
+            Caster::PREFIX_VIRTUAL.'pattern' => $c->getPattern(),
+            Caster::PREFIX_VIRTUAL.'attributes' => new EnumStub(
```

Comment:
before adding this key to the array, I'd suggest adding its last `, $filter = 0` argument to the method and using it in a block like:
```
if ($filter & Caster::EXCLUDE_VERBOSE) {
    $stub->cut += 3;

    return $a;
}
```